package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "open-msupply",
+  "version": "1.1.0",
```

Comment:
As per our convo in standup, going up to 1.1.0 allows for template migrations (see upstream branch), to be compiled and tested easier